experimental/PropertyDDS/packages/property-properties/src/properties/abstractStaticCollectionProperty.js

```diff
@@ -347,10 +347,11 @@ export class AbstractStaticCollectionProperty extends BaseProperty {
 	 * Given an object that mirrors a PSet Template, assigns the properties to the values
 	 * found in that object.
 	 * See {@link setValues}
-	 * @param {object} in_values - The object containing the nested values to assign
+	 * @param {Object | Array | string} in_values - The object containing the nested values to assign
```

Comment:
Base class signatures must accept a superset of types accepted by overrides.

In this case, '_setValues' is overriden by MapProperty to accept an Object, ArrayProperty to accept an Array, and StringProperty to accept a string.